contrib/for-tests/mount-tester-user/Makefile

```diff
@@ -0,0 +1,9 @@
+all: push
+
+TAG = 0.1
+
+image:
+	sudo docker build -t kubernetes/mounttest-user:$(TAG) .
+
+push: image
+	sudo docker push kubernetes/mounttest-user:$(TAG)
```

Comment:
s/kubernetes/gcr.io\/google_containers/ in this file. Also can you make the sudo in image be gcloud like:

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/contrib/for-tests/mount-tester/Makefile#L12

This will make it easier for the next person to push the image.